templates/bot.mustache

```diff
@@ -9,8 +9,14 @@
     {{> bot_css }}
 </head>
 
+<script>
+    const host_name = $(location).attr('hostname');
+    const isBinaryDomain = host_name !== 'localhos' && host_name !== 'bot.binary.com';
```

Comment:
i think you would  like to write 'localhos**t**' instead  of 'localhos'

Reply:
It's just a chore. Removed, thanks